.github/workflows/build.yml

```diff
@@ -53,6 +53,30 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Extract branch name
+        id: branch_env
+        shell: bash
+        run: |
+          export VERSION=${GITHUB_REF##*/}
+          echo "##[set-output name=version;]$(echo ${VERSION:1})"
```

Comment:
Like currently when we run on PR, the last word of `GITHUB_REF` would be `merge`. When it runs when push, it would become the branch name. And when it runs on release branch, branch like `v1.5` would become `1.5`.